src/styles/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 16px;
+
+  &__product-image {
+    padding: 16px 3px 0;
+    display: flex;
+    top: 33px;
+    left: 20px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__product-name {
+    margin: 0;
+    padding-top: 40px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
+  }
+
+  &__product-code {
+    margin: 0;
+    padding-top: 4px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+  }
+
+  &__product-rating {
+    padding-top: 16px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    width: 166px;
+    height: 16px;
+
+    &-review {
+      margin: 0;
+      display: flex;
+      font-weight: 400;
+      font-size: 10px;
+      line-height: 14px;
+      color: #060b35;
+      align-self: flex-end;
+    }
+  }
+
+  &__product-price {
+    padding-top: 24px;
+    padding-bottom: 16px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    line-height: 18px;
+
+    &-text {
+      margin: 0;
+      font-weight: 400;
+      font-size: 12px;
+      color: #616070;
+    }
+
+    &-numbers {
+      margin: 0;
+      font-weight: 700;
+      font-size: 16px;
+    }
+  }
+
+  &__product-buy-link {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border-radius: 5px;
+    width: 166px;
+    height: 40px;
```

Comment:
The width of the button should be controlled by the parent container, not set directly on the button. It's better to set width as 100% so that it takes the full width of its parent.